DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -7,8 +7,10 @@ const route = `TrackingSystem/Delegates/Email/AllEmailDelegateItems?${queryParam
 
 // eslint-disable-next-line no-new
 new SearchSortFilterAndPaginate(route, false, false, true, 'EmailDelegateFilter');
+setupSelectAndDeselectButtons();

```

Comment:
Super minor, but given this is a verb, it should technically be "setUp" re capitalisation 👁️